setup-buildengine.sh

```diff
@@ -46,7 +46,7 @@ OPT_COMPRESSION=9
 OPT_RELEASE="latest"
 OPT_TOOLING="Jolla-latest-Sailfish_SDK_Tooling-i486.tar.bz2"
 OPT_TARGET_ARM_ARMV7HL="Jolla-latest-Sailfish_SDK_Target-armv7hl.tar.bz2"
-OPT_TARGET_I486="Jolla-latest-Sailfish_SDK_Target-i486.tar.bz2"
+OPT_TARGET_X86_I486="Jolla-latest-Sailfish_SDK_Target-i486.tar.bz2"
```

Comment:
Like the extra ARM in OPT_TARGET_ARM_ARMV7HL is redundant, the X86 in X86_I486 is also redundant.